### ISSUE TITLE
How to add it to nvim that doesn't have Lazy.nvim?

### ISSUE THREAD

**Issue by sergeda** [you-n-g/navigate-note.nvim#2]
I'm using pckr plugin for installation of the plugins.
I added 
```
{ "you-n-g/navigate-note.nvim",
 config = function() 
   require('navigate-note').setup() 
 end 
};
```
Plugin installed fine, but when I start Neovim I'm getting:
```
[pckr.nvim[ERROR 15:57:58] loader.lua:10: Error running config for navigate-note.nvim:                                                                                        
^I...ckr/opt/navigate-note.nvim/lua/navigate-note/keymaps.lua:17: attempt to call field 'add' (a nil value)  
```

**Comment by you-n-g** [comment]
I have add a pckr test just now. 
- https://github.com/you-n-g/navigate-note.nvim?tab=readme-ov-file#development  
- corresponding commit https://github.com/you-n-g/navigate-note.nvim/commit/920adc32c4c9378cde4f7d8470b00366027473d1

It works on my case.



Based on your message, I guess that you may have installed a different plugin also named `which-key`.  
If I am right, I have fixed it in [f454785](https://github.com/you-n-g/navigate-note.nvim/commit/f454785e25e7872e60856cb84f0695869008e37c). 
So try to update to the latest version to fix this issue.

**Comment by sergeda** [comment]
Nice. Updating fixed the issue. Thank you
